pages/linux/abroot.md

```diff
@@ -0,0 +1,37 @@
+# abroot
+
+> ABRoot utility provides full immutability and atomicity by transacting between 2 root partition states (A⟺B).
+> It also allows on-demand transactions via a transactional shell.
+> More information: <https://github.com/Vanilla-OS/ABRoot>.
+
+- Output the present root partition state:
+
+`sudo abroot get present`
+
+- Output the future root partition state:
+
+`sudo abroot get future`
+
+- Enter the transactional shell in the future root partition and switch root on the next boot:
+
+`sudo abroot shell`
+
+- Execute a command in the transactional shell in the future root partition and switch to it on the next boot:
+
+`sudo abroot exec {{command}}`
```

Comment:
> if [...] `exec` subcommand expect[s] the whole command as [s] single string.

Do you know whether that is the case?

Reply:
> Do you know whether that is the case?

No, I just assume. That's why I guess that PR author knows better than me and just show what can be done if my assumption is right.

Reply:
> Optional fix: placeholder can be quoted if it can contain of several space delimited terms and `exec` subcommand expect the whole command as single string.

Interesting I will test it out before committing the change.